src/front/app/components/globals/Label.js

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+
+class Label extends Component {
+
+   static propTypes = {
+    children: PropTypes.element.isRequired,
+    className: PropTypes.string
+  };
+
+
+  render() {
+    const { className, children } = this.props;
+
+
```

Comment:
Ideally, set the classes like this (rather than inside className):
```js
const classes = classNames({
    [className]: !!className,
});
```

this so that if we want to add some default label classes (which we will probably end up wanting) its easier to add.